### ISSUE TITLE
amostra

### ISSUE THREAD

**Issue by Ionneto** [Ionneto/ds-floyd#1]
@ranisalt

help, tem como amostrar o que ta no .tex na home do repo? acho que ouvi alguem do pet falando sobre isso esses tempos...

**Comment by Ionneto** [comment]
@matheusben-hur

**Comment by ranisalt** [comment]
Não, não dá porque o GitHub não sabe interpretar TeX. O melhor que dá de fazer é usar markdown mesmo.
